docs/whirlpool/docs/03-Getting Started/01-Environment Setup.md

```diff
@@ -0,0 +1,135 @@
+# Environment setup
+
+This document covers the essential setup required to start building on Orca’s SDK using the Whirlpools protocol. It includes wallet setup, RPC client configuration, airdropping tokens for testing, and the basics of interacting with the Solana ecosystem.
+
+## Prerequisites
+
+Before you start, ensure you have Node.js installed on your machine. Download it from the official website: https://nodejs.org/.
+
+## 1. Initialize a new project
+Create a new project directory:
+
+```bash
+mkdir whirlpool-env-setup
+cd whirlpool-env-setup
+```
+
+Initialize a new Node.js project:
+
+```bash
+npm init -y
+```
+
+Install the necessary packages:
+
+```bash
+npm install typescript ts-node @orca-so/whirlpools-client @solana/web3.js fs
```

Comment:
The high level sdk is `@orca-so/whirlpools`

Reply:
Do we need ts-node and fs here? The easier we can keep this the easier it becomes for a reader to follow along